lib/ts/recipe/emailpassword/types.ts

```diff
@@ -246,6 +256,34 @@ export type APIInterface = {
               | {
                     status: "EMAIL_ALREADY_EXISTS_ERROR";
                 }
+              | {
```

Comment:
add a boolean in status: OK called `createdNewUser`. See https://github.com/supertokens/supertokens-core/issues/447#issuecomment-1298433805